indra/lib/python/indra/util/llmanifest.py

```diff
@@ -815,8 +820,8 @@ def try_path(src):
             # expand globs
             count = 0
             if self.wildcard_pattern.search(src):
-                for s,d in self.expand_globs(src, dst):
-                    assert(s != d)
+                for s, d in self.expand_globs(src, dst):
+                    assert (s != d)
```

Comment:
Why space here?

Reply:
Style guide says so: https://pep8.org

Reply:
After assert but before parentheses?
So then assert is just a keyword and not a function call?
Maybe we should remove the parentheses around the condition, then, since they're not doing anything?

Reply:
Yes, `assert` in Python is a "simple statement" not a function call. `assert expression[, assertion_message]`
I'd just leave the expression in parenthesis as is, but they could be removed.

Reply:
Yeah, it's fine enough as is.